src/packages/src/xroad/ubuntu/generic/xroad-center.config

```diff
@@ -22,15 +22,22 @@ if [ "$1" = "configure" ] || [ "$1" = "reconfigure" ]; then
   done
 
   # Get database connection string from the user
-  db_host=$(get_prop /etc/xroad/db.properties host "")
-  db_port=$(get_prop /etc/xroad/db.properties port 5432)
+  db_host=$(get_prop /etc/xroad/db.properties host)
+  db_port=$(get_prop /etc/xroad/db.properties port)
```

Comment:
the default value for the port seems to be necessary, since if it is missing (and host is set) the db_set on line 28 produces an invalid value.

Reply:
Fixed in https://github.com/nordic-institute/X-Road/pull/1024/commits/3fe01adb46e8dcb0b3598977262360ebbe0dd89c